packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -698,14 +700,13 @@ class DebuggerController extends DisposableController
           variable.addAllChildren(_createVariablesForAssociations(result));
         } else if (result.elements != null) {
           variable.addAllChildren(_createVariablesForElements(result));
+        } else if (result.bytes != null) {
```

Comment:
This needs to be above the fields check, since fields doesn't seem to ever be null, just an empty list

Reply:
Done